services/blockchain-indexer/shared/dataService/business/transactions.js

```diff
@@ -101,6 +103,7 @@ const validateParams = async params => {
 
 const getTransactions = async params => {
 	const transactionsDB = await getTransactionsIndex();
+	const blocksTable = await getBlocksIndex();
```

Comment:
Can we please update it to `blocksDB` to be consistent
```suggestion
	const blocksDB = await getBlocksIndex();
```